<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>Thomas Guillerme</title>
    <link rel="stylesheet" href="stylesheets/styles.css">
    <link rel="stylesheet" href="stylesheets/pygment_trac.css">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>
    <script src="javascripts/respond.js"></script>
    <!--[if lt IE 9]>
      <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
    <!--[if lt IE 8]>
    <link rel="stylesheet" href="stylesheets/ie.css">
    <![endif]-->
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">



<!-- Add the whole idea of "from tree inference to tree analysis" -->

  </head>
  <body>
      <div id="header">
        <nav>
          <li class="fork"><a href="index.html">Home</a></li>
          <li class="fork"><a href="research.html">Research</a></li>
          <li class="fork"><a href="publications.html">Publications</a></li>
          <li class="fork"><a href="cv.html">CV</a></li>
          <li class="fork"><a href="outreach.html">Outreach</a></li>
          <li class="fork"><a href="https://github.com/TGuillerme">GitHub</a></li>
        </nav>
      </div><!-- end header -->

    <div class="wrapper">

      <section>
        <div id="title">
          <p><img src="images/PrimatesVar.jpg" height="200"/> 
            <span> <h1>Research</h1> </span>
          </p>
          <hr>
        </div>

        <h2>
          <a name="How does including fossils" class="anchor" href="#includingfossils"><span class="octicon octicon-link"></span></a>Macroevolution with fossil and living taxa
          <!-- How does including fossils into phylogenetic trees changes our interpretation of phylogenetic comparative methods?-->
        </h2>
        <h3> <center> <i>From phylogenetic inference to comparative methods</i> </center> </h3>

        <p>
          I develop and implement new methods allowing to use morphological data to answer macroevolutionary questions using both fossil and living taxa.
          My work ranges from improving models in phylogenetic inference to comparative methods.
          I am particularly interested in using morphology in modern phylogenetics and macroevolutionary methods.
        </p>




        <h3><center>Morphological characters in phylogenetics</center></h3>
        <center><a href="http://www.slideshare.net/ThomasGuillerme/total-evidence-phylogenies-the-missing-data-issue"><img src="images/252525Missing.jpg" height="200"/></a> </center>
        <p><center><i>Cladistic, Total Evidence method, Tip-dating, missing data, M<i>k</i> model, Bayesian inference, inapplicable data, character correlation.</i></center></p>


        <!-- CENTER GALLERY0 -->

   <ul id="gallery1">
          <li>
            <p><a href="https://github.com/TGuillerme/Inapp"><img src="images/252525Missing.jpg" height="60" widht="60"
                 alt="" />Inapplicable data in phylogenetics</a></p>
          </li>

          <li>
            <p><a href="https://github.com/TGuillerme/Total_Evidence_Method-Missing_data"><img src="images/GitHub.png" height="60" widht="60"
                 alt="" />Missing data in Total Evidence</a></p>
          </li>

          <li>
            <p><a href="https://github.com/TGuillerme/CharactersCorrelation"><img src="images/GitHub.png" height="60" widht="60"
                 alt="" />Character correlation in phylogenetics</a></p>
          </li>
    </ul><!-- end gallery0 -->

        <p>
          The last five years have witnessed increased enthusiasm for using both living and fossil data to better understand macroevolutionary patterns and processes. This is partially due to recent and successful implementations of phylogenetic software (e.g. MrBayes, BEAST) that can deal with both morphological (cladistic) and molecular data. Our understanding of molecular evolution has remarkably enhanced in the last two decades leading to excellent knowledge in both the mechanisms underlying DNA evolution as well as the software implementations to compute it efficiently. Unfortunately, our understanding of more complex morphological character evolution is still lacking and require both theoretical and practical improvements.
        </p>
        <p>
          I am interested in exploring the importance of morphological characters in the area of full genome sequencing and how this data allows us to better understand macroevolution. I tackle questions such as <b>the effect of missing morphological data</b> in such analysis as well as trying to <b>develop more accurate probabilistic models to better estimate morphological characters evolution</b>.
        </p>



        <h3><center>Disparity through time</center></h3>
        <center><a href="https://figshare.com/articles/New_approaches_to_disparity-through-time_analysis/3437546"><img src="images/252525dispRity.jpg" height="200"/></a> </center>
        <p><center><i>Disparity metrics, time-slicing, evolutionary models, multidimensional statistics</i></center></p>


   <ul id="gallery1">
          <li>
            <p><a href="https://github.com/TGuillerme/dispRity"><img src="images/Rlogo.png" height="60" widht="60"
                 alt="" />dispRity: measuring disparity</a></p>
          </li>

          <li>
            <p><a href="https://github.com/nhcooper123/time-slice"><img src="images/252525dispRity.jpg" height="60" widht="60"
                 alt="" />Time slicing!</a></p>
          </li>
    </ul><!-- end gallery0 -->

        <p>
          Morphological data is mainly used for solving relationships among lineages and understanding timing of clade evolution. However, another emergent property from this data is that it also allows to look at the evolution of the raw morphology through time for answering macroevolutionary question such as the effect of mass extinction or large scale competition on morphological evolution. Palaeontological data allows us to look at the distribution of this morphological data through time in a hyper-dimensional way often summarised under the umbrella term <i>disparity analysis</i> and how specific lineages evolve through this hyper-dimensional object often designed as the <i>morphospace</i>
        </p>
        <p>
          Disparity can be seen as the summary of the morphospace, however, not much is known on which metric to use for capturing which specific aspect (typically change) of the morphospace. Also, methods to estimate the placement of lineages through time in this morphospace is still crude and does not allow evolutionary models variation. I am interested in understanding <b>which disparity metric summarises which aspect of the morphospace</b> as well as developing a new method of looking a <b>disparity through time in a continuous fashion allowing the use of different evolutionary models</b>.
        </p>



          <!-- Link to dispRity, disparity projects? -->

        <h2>Macroecology</h2>
        <h3><center>Using phylogenetic uncertainty in comparative methods</center></h3>
        <center><a href="https://figshare.com/articles/Guillerme_BESMacro2016_pdf/3478922"><img src="images/252525mulTree.jpg" height="100"/></a> </center>
        <p><center><i>Phylogenetic correction, MCMCglmm, Bayesian posterior tree distribution</i></center></p>


   <ul id="gallery1">
          <li>
            <p><a href="https://github.com/TGuillerme/mulTree"><img src="images/Rlogo.png" height="60" widht="60"
                 alt="" />mulTree: tree uncertainty in PCM</a></p>
          </li>

          <li>
            <p><a href="https://github.com/healyke/SIDER"><img src="http://healyke.github.io/images/SIDER.png" height="60" widht="60"
                 alt="" />SIDER: isotope analysis and PCM</a></p>
          </li>
    </ul><!-- end gallery0 -->

        <p>
          Classic macroecological studies uses phylogenetic comparative methods to account for the fact that macroevolution played a key role in the observed ecological patterns today. However, many macroecologists imply that phylogenetic patterns are known without uncertainty and usually uses only one consensus phylogenetic tree in their analysis. Even though phylogenetic inference have tremendously advanced since the last two decades, some phylogenetic trees still come with much uncertainty on nodes and branch length. The Bayesian posterior tree distribution allows us to directly access this uncertainty (or <i>noise</i>) in phylogenetic inference.
        </p>
        <p>
          I am interested in <b>developing new algorithms that allows to use this phylogenetic uncertainty in comparative methods</b>especially by using the Bayesian posterior tree distribution in advanced and popular comparative methods such as general linear mixed models (glmm). 
        </p>


        <h2>Didn't understood a word?</h2>
        <h3><center>Read my <a href="http://splasho.com/upgoer5/">Up-Goer 5</a> research:</center></h3>
        <p>
          I am interested in looking how animals forms changed through time. I use computers to make not real (but real looking!) changes of animals forms through time to understand how the things (like the death of many animals or when a group of animal try to fight another one) can change the forms of the animals or even allow new animal forms to appear. This also allows me to look if groups of animals are in the same family and how their family changed through time. 
        </p>


      </section>
    </div>
    <!--[if !IE]><script>fixScale(document);</script><![endif]-->
    
  </body>
</html>
